stable/pgadmin/templates/deployment.yaml

```diff
@@ -54,13 +54,10 @@ spec:
             {{- .Values.readinessProbe | toYaml | nindent 12 }}
         {{- end }}
           env:
-            - name: PGADMIN_DEFAULT_EMAIL
-              value: {{ .Values.env.email }}
-            - name: PGADMIN_DEFAULT_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: {{ $fullName }}
-                  key: password
+            {{- range $pkey, $pval := .Values.env }}
+            - name: {{ $pkey }}
+              value: {{ quote $pval }}
+            {{- end }}
```

Comment:
Please use the old way of defining these variables, so it is easier to read.

e.g.:
```
env:
  - name: PGADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION
    value: {{ .Values.env.enhanced_cookie_protection }}
```

Reply:
yeh effectively it will be more clear to read but if someone want to use a new env he will be obligate to modifies the chart :)